Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.h

```diff
@@ -67,7 +67,7 @@ class ApplicationManifestParser {
     void logManifestPropertyInvalidURL(const String&);
     void logDeveloperWarning(const String& message);
 
-    RefPtr<Document> m_document;
+    const RefPtr<Document> m_document;
```

Comment:
While you're modifying this file, do you mind modifying the constructor and use `RefPtr<Document>&&` for the parameter instead of `RefPtr<Document>` and then use `WTFMove()` internally?